TM1py/Services/CellService.py

```diff
@@ -330,7 +362,7 @@ def write_dataframe(self, cube_name: str, data: 'pd.DataFrame', dimensions: Iter
         cells = build_cellset_from_pandas_dataframe(data)
 
         return self.write_values(cube_name=cube_name, cellset_as_dict=cells, dimensions=dimensions,
```

Comment:
I think it makes sense to call the new `write` function here instead. Then we could expose the `use_ti` argument also to the `write_dataframe` function.
What do you think?

Reply:
Yes, that makes sense to me that would be better